src/Components/Layout/Navigation.js

```diff
@@ -0,0 +1,21 @@
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
do you have to use this?

Reply:
Yes, I tried without, but this styling is required for Bootstrap components

Reply:
ok awesome.